src/internal.c

```diff
@@ -12903,6 +12930,18 @@ static int ProcessPeerCertParse(WOLFSSL* ssl, ProcPeerCertArgs* args,
     }
 #endif
 
+#if defined(WOLFSSL_MAXQ10XX_TLS) && defined(HAVE_PK_CALLBACKS)
```

Comment:
I'm okay with enabling this PK call the time, but because of the `DecodedStruct` it would need to be gated on `WOLFSSL_PUBLIC_ASN`